unicode_smilies/unicode_smilies.php

```diff
@@ -1191,11 +1015,20 @@ function unicode_smilies_smilies(&$a,&$b) {
 	Smilies::add($b, ':skis:', '&#x1F3BF');
 	Smilies::add($b, ':sled:', '&#x1F6F7');
 	Smilies::add($b, ':curling stone:', '&#x1F94C');
+    Smilies::add($b, ':cycling:', '&#x1F6B4;');
```

Comment:
Code standards: Please normalize indentation.
```suggestion
	Smilies::add($b, ':cycling:', '&#x1F6B4;');
```